README.md

```diff
@@ -56,15 +56,25 @@ releases of the toolkit.
     * Python 3.6.2, along with a set of additional Python packages, is required to run some tools and workflows.
       GATK uses the [Conda](https://conda.io/docs/index.html) package manager to establish and manage the
       environment and dependencies required by these tools. The GATK Docker image comes with this environment
-      pre-configured. In order to establish an environment suitable to run these tools outside of the Docker image, the
-      conda [gatkcondaenv.yml](https://github.com/broadinstitute/gatk/blob/master/scripts/gatkcondaenv.yml) file is
-      provided. To establish the conda environment locally, [Conda](https://conda.io/docs/index.html) must first
-      be installed. Then, create the gatk environment by running the command ```conda env create -n gatk -f gatkcondaenv.yml```
-      (developers should run ```./gradlew createPythonPackageArchive```, followed by
-      ```conda env create -n gatk -f scripts/gatkcondaenv.yml``` from within the root of the repository clone).
-      To activate the environment once it has been created, run the command ```source activate gatk```. See the
-      [Conda](https://conda.io/docs/user-guide/tasks/manage-environments.html) documentation for
-      additional information about using and managing Conda environments.
+      pre-configured. In order to establish an environment suitable to run these tools outside of the Docker image,
+      do the following:
+        * Make sure [Conda or MiniConda](https://conda.io/docs/index.html) is installed. Miniconda is sufficient.
```

Comment:
Split out a separate section on `conda` in the README, and link to it from this Requirements bullet, instead of having it all inline.

Reply:
This also needs to mention the Intel environment. @erniebrau we want to start shipping a conda environment yml for that right ?